packages/i18n/README.md

```diff
@@ -27,6 +27,14 @@ For a complete example, see the [Internationalization section of the Block Edito
 
 <!-- START TOKEN(Autogenerated API docs) -->
 
+<a name="isRTL" href="#isRTL">#</a> **isRTL**
+
+Check if current locale is RTL.
```

Comment:
Explaining what "RTL" means could be good here.

Reply:
This is the exact documentation string from `WP_Locale::is_rtl()`, but you're right 🙂 

I'll add some detail.

Reply:
Added RTL explanation courtesy of https://developer.mozilla.org/en-US/docs/Glossary/rtl.